fs2-aws/src/main/scala/fs2/aws/internal/package.scala

```diff
@@ -28,7 +27,7 @@ package object internal {
   )(implicit F: Concurrent[F]): Pipe[F, A, (K, Stream[F, A])] = { in =>
     Stream.eval(Ref.of[F, Map[K, Queue[F, Option[A]]]](Map.empty)).flatMap { queueMap =>
       val cleanup = {
-        queueMap.get.flatMap(_.traverse_(_.enqueue1(None)))
+        queueMap.get.flatMap(_.values.toList.traverse_(_.enqueue1(None)))
```

Comment:
@semenodm  This stopped compiling.  I think it _might_ have been to do with the move to 2.12.10 - Either way alleycats is deprecated.  I hope this is an equivalent.